examples/tests/keyboard_hid/main.c

```diff
@@ -0,0 +1,35 @@
+#include <string.h>
+
+#include <button.h>
+#include <usb_keyboard_hid.h>
+
+
+uint8_t keyboard_buffer[64];
```

Comment:
This 64 should be a macro

Reply:
I don't see what the extra layer of indirection accomplishes.

Reply:
Because you do this later

```c
err = usb_keyboard_hid_send_sync(buffer, 64);
```

If someone changes the size here that code will be incorrect. If you use a macro then the size of `keyboard_buffer` and the length supplied to functions will always match.